src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -168,7 +168,7 @@ For example, `sub-01_mod-T1w_defacemask.nii.gz`.
 If several scans of the same modality are acquired they MUST be indexed with the
 [`run-<index>`](../99-appendices/09-entities.md#run) key-value pair:
 `_run-1`, `_run-2`, `_run-3`, and so on (only nonnegative integers are allowed as
-run labels). When there is only one scan of a given type the run key MAY be
+run labels). If however these scans of the same modality contain different labels (e.g. `_acq-`, `_ce-`, `_desc-`) then the run label and indexing are optional, unless the labels are the same. When there is only one scan of a given type the run key MAY be
```

Comment:
Thanks for the PR @dlevitas I suggest to break lines at reasonable places to improve future git diff, and to replace "e.g." by more readable "for example", because we are trying to reduce the use of latin in the BIDS specification where possible.

```suggestion
run labels).
If however these scans of the same modality contain different labels
(for example `_acq-`, `_ce-`, `_desc-`) then the run label and indexing are optional,
unless the labels are the same.
When there is only one scan of a given type the run key MAY be
```